### DISCUSSION TITLE
GIMP Plug-in to quickly upscale using RESRGAN

### DISCUSSION THREAD

**Discussion by Nenotriple** [xinntao/Real-ESRGAN#856] [Show and tell]
Hello, I'd like to share a simple GIMP plug-in I made that uses the portable NCNN version.

You can install this plug-in and quickly upscale images directly inside GIMP.

Please see my GitHub page for more details: https://github.com/Nenotriple/gimp_upscale

**Comment by FlM122** [comment]
which model is the best for anime videos??? to ensure that the quality is as good as possible and that the speed is high . I found AnimeJaNai, but it makes the contour lines too sharp.
